packages/tic-tac-toe/app/routes/application.ts

```diff
@@ -0,0 +1,23 @@
+import Route from '@ember/routing/route';
+import Transition from '@ember/routing/-private/transition';
+import {inject as service} from '@ember/service';
+import {ChannelClient} from '@statechannels/channel-client';
+
+import {ChannelProviderInterface} from '@statechannels/channel-provider';
+import TttChannelClientService from '../services/ttt-channel-client';
+
+declare global {
+  interface Window {
+    channelProvider: ChannelProviderInterface;
+  }
+}
+
+export default class ApplicationRoute extends Route {
+  @service tttChannelClient!: TttChannelClientService;
+
+  async beforeModel(transition: Transition): Promise<void> {
+    super.beforeModel(transition);
+    window.channelProvider.enable('http://localhost:3055/');
```

Comment:
maybe this could be set on an environment variable?

Reply:
Yes it should be. I just haven't done this yet. I will open an issue for this.